opencog/reasoning/pln/rules/and-simplification-rule.scm

```diff
@@ -0,0 +1,51 @@
+; =====================================================================
+; AndSimplificationRule
+;
+; AndLink 
+;    A
+;    AndLink
+;       B
+;	C
+; |-
+; AndLink
+;    A
+;    B
+;    C
+;----------------------------------------------------------------------
+
+
+(define pln-rule-and-simplification
+  (BindLink
+   (VariableList
+    (VariableNode "$A")
+    (VariableNode "$B")
+    (VariableNode "$C"))
+   (AndLink
+     (VariableNode "$A")
+     (AndLink
```

Comment:
It would be best if the AndLink is a VarialbleNode restricted to an AndLink in the variable declartion. That way it would work with n-ary AndLink